src/devices/Gpio/Drivers/OrangePiLite2Driver.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for the Orange Pi Lite 2.
+    /// </summary>
+    /// <remarks>
+    /// SoC: Allwinner H6 (sun50iw6p1)
+    /// </remarks>
+    public class OrangePiLite2Driver : Sun50iw6p1Driver
```

Comment:
IMO those drivers should be internal and we should change detection logic, on the other hand we did other drivers public. I feel like this library will grow in size indefinitely if we keep going like that. Perhaps we should split the drivers into separate package or something to only pull what's needed on the given setup? cc: @Ellerbach @joperezr